ibis/backends/dask/execution/generic.py

```diff
@@ -401,6 +403,27 @@ def wrap_case_result(raw: np.ndarray, expr: ir.ValueExpr):
     return result
 
 
+@execute_node.register(ops.SearchedCase, list, list, object)
+def execute_searched_case_dask(op, whens, thens, otherwise, **kwargs):
+    if not isinstance(whens[0], dd.Series):
```

Comment:
When would you have the case that this is not a dd.Series?

Reply:
https://github.com/ibis-project/ibis/blob/master/ibis/backends/dask/tests/execution/test_operations.py#L931

Reply:
Can you add a comment when this is dd.Series vs when it is not? Also for this example:

```
x.case().when(2, x - 1).when(3, x + 1).when(4, x + 2).end()
```

if x is a column, then are we still doing the right thing calling the panda execute?

Reply:
Sorry - ignore my original comment. 

```python
expr = ibis.case().when(True, 1).when(False, 2).end()
```
That is the code that will go through the pandas branch (I've added a pointer to this in the code). 

For your case, that actually gets handled by a different function (`execute_simple_case_series`). For both pandas and dask the following will return `nan`s:

```
    x = t.float64_with_zeros
    expr = x.case().when(2, x - 1).when(3, x + 1).when(4, x + 2).end()
```

Reply:
kk